lib/veritaserum.ex

```diff
@@ -27,44 +27,69 @@ defmodule Veritaserum do
       3
   """
   @spec analyze(String.t) :: Integer.t
-  def analyze(input) do
-    input
-    |> clean
-    |> String.split
-    |> split_on_emoticons
-    |> Enum.map(&String.trim/1)
-    |> analyze_list()
-    |> Enum.reduce(0, &(&1 + &2))
-  end
 
-  defp analyze_list([head | tail]) do
-    analyze_list(tail, head, [analyze_word(head)])
+  def analyze(input, options \\ [])
+  def analyze(input, options) do
+    return = Keyword.get(options, :return, :score)
+
+    marked_list = input
+                  |> clean
```

Comment:
Would be great if we could indent this like

```elixir
marked_list = input
  |> clean
  |> String.split
  |> ...
```

The same for the pipes below :)